### ISSUE TITLE
Where are the C++ communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/cpp#181]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is C++ supported by one or more large organizations?
* Does C++ have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about C++? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for C++? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in C++? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in C++?
* Are there popular newsletters for C++?
* Is C++ taught at programming bootcamps? (If so, what are some examples?)
* Is C++ taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about C++?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by LegalizeAdulthood** [comment]
* Is C++ supported by one or more large organizations?
  * There are large companies that use C++: NVIDIA, Microsoft, Google, Apple, Bloomberg, etc., and they sometimes sponsor conferences and other events, but I don't know if that is considered to be sponsoring the language itself.  C++ is used in billions of devices worldwide in everything from the smallest embedded CPU to the largest supercomputer.
* Does C++ have an official community manager?
  * Nope.  The closest thing I can think of would be the [Microsoft MVPs for C++](https://mvp.microsoft.com/en-us/Pages/mvp-award-update) and even those are not community managers, just helpful members of the community.  Is there a FORTRAN community manager?  A LISP community manager?  These languages are not associated with a specific organization or company and there is noone to pay the salary of a community manager and the community is too disparate among the many disciplines that use the language to even attempt to "manage" them.
* Do you know of specific communities (online or offline) that are enthusiastic about C++? (Chat communities, forums, meetups, student clubs, etc)
  * [cplusplus.com forums](http://www.cplusplus.com/forum/)
  * [cpplang slack](https://cpplang.now.sh/)
  * [Worldwide user groups listing](https://isocpp.org/wiki/faq/user-groups-worldwide)
* Are there popular conferences for C++? (If so, what are some examples?)
  * [CppCon](https://cppcon.org/)
  * [PacifiC++](https://pacificplusplus.com/)
  * [C++ Now!](http://cppnow.org/)
  * [Cpp Europe](https://cppeurope.com/)
  * [Worldwide conference listing](https://isocpp.org/wiki/faq/conferences-worldwide/)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in C++? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
  * Probably, but I don't know of any specifically
* Are there specific groups or programs dedicated to mentoring people in C++?
  * Probably, but I don't know of any specifically
* Are there popular newsletters for C++?
  * Probably, but I don't know of any specifically
* Is C++ taught at programming bootcamps? (If so, what are some examples?)
  * Probably, but I don't know of any specifically
* Is C++ taught at universities? (If so, what are some examples?)
  * Yes, but it depends on the University.  University of Utah uses C++ in [CS 1410: Intro to Object-Oriented Programming](http://www.cs.utah.edu/~parker/1410_030/syll.pdf)
  * It is highly likely that almost every STEM oriented school in the United States, United Kingdom and Europe has some course in their curricula that teaches or uses C++.  There are probably lots more beyond that description.  Every artist using [Arduino](https://en.wikipedia.org/wiki/Arduino) or similar platforms is using C++, for instance.

**Comment by kytrinyx** [comment]
This is very helpful, thank you!

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you!

**Comment by KantarBruceAdams** [comment]
Just passing by but I noticed you missed isocpp.org. This is acting as a hub to bring some of the disparate communities together as well as tracking the ISO standard.

**Comment by patricksjackson** [comment]
@KantarBruceAdams It's not mentioned here, but we do have it listed in the track docs/RESOURCES.md.

**Comment by malikfayaz1** [comment]
C++ Programming Step By Step From Beginner To Ultimate Level 
https://www.udemy.com/course/c-programming-step-by-step-from-beginner-to-ultimate-level/?couponCode=LEARNNOW1
